src/tests/CommonTestRunner/TestRunnerUtilities.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Diagnostics.TestHelpers;
+using Xunit.Abstractions;
+using TestRunner = Microsoft.Diagnostics.CommonTestRunner.TestRunner;
+
+namespace CommonTestRunner
+{
+    public static class TestRunnerUtilities
+    {
+        public static async Task<TestRunner> StartProcess(TestConfiguration config, string testArguments, ITestOutputHelper outputHelper, int testProcessTimeout = 60_000)
+        {
+            TestRunner runner = await TestRunner.Create(config, outputHelper, "EventPipeTracee", testArguments).ConfigureAwait(true);
```

Comment:
Can we consolidate this with [src/tests/Microsoft.Diagnostics.Monitoring.EventPipe/PipelineTestUtilities.cs](https://github.com/dotnet/diagnostics/blob/369e872ba653550c875b1a421cf3b718707249ff/src/tests/Microsoft.Diagnostics.Monitoring.EventPipe/PipelineTestUtilities.cs#L19)